src/v2/Components/SavedSearchAlert/Utils/__tests__/convertLabelsToPills.jest.ts

```diff
@@ -0,0 +1,74 @@
+import { convertLabelsToPills } from "../convertLabelsToPills"
+
+describe("convertLabelsToPills", () => {
+  it("should return correctly formatted pills", () => {
+    const result = convertLabelsToPills([
+      {
+        field: "attributionClass",
+        displayValue: "Unknown edition",
+        value: "unknown edition",
+      },
+      {
+        field: "additionalGeneIDs",
+        displayValue: "Photography",
+        value: "photography",
+      },
+    ])
+
+    expect(result).toEqual([
+      {
+        isDefault: false,
+        field: "attributionClass",
+        value: "unknown edition",
+        displayValue: "Unknown edition",
+      },
+      {
+        isDefault: false,
+        field: "additionalGeneIDs",
+        value: "photography",
+        displayValue: "Photography",
+      },
+    ])
+  })
+
+  it("should correctly formatted default pills", () => {
+    const result = convertLabelsToPills([
+      {
+        field: "artistIDs",
+        displayValue: "Banksy",
+        value: "4dd1584de0091e000100207c", // pragma: allowlist secret
+      },
+      {
+        field: "artistIDs",
+        displayValue: "KAWS",
+        value: "4e934002e340fa0001005336", // pragma: allowlist secret
+      },
+      {
+        field: "sizes",
+        displayValue: "Medium (40 – 100cm)",
+        value: "MEDIUM",
+      },
+    ])
+
+    expect(result).toEqual([
+      {
+        isDefault: true,
+        field: "artistIDs",
+        value: "4dd1584de0091e000100207c", // pragma: allowlist secret
```

Comment:
**question**: Interesting. Does `detect-secrets` think that this is a secret otherwise?

Reply:
Yes, `detect-secrets` thinks that this is some kind of secret information